hack/docker/voyager/templates/tcp-frontend.cfg

```diff
@@ -1,4 +1,9 @@
 frontend {{ .FrontendName }}
 	bind *:{{ .Port }} {{ if .AcceptProxy }}accept-proxy{{ end }} {{ if .SecretName }}ssl no-sslv3 no-tlsv10 no-tls-tickets crt /etc/ssl/private/haproxy/{{ .SecretName }}.pem{{ end }} {{ if .ALPNOptions }}{{ .ALPNOptions }}{{ end }}
 	mode tcp
+
+	{{- range $rule := .FrontendRules }}
```

Comment:
Fix indentation

Reply:
blank line will be removed from the generated config.